test/src/template/array_selector_test.dart

```diff
@@ -63,8 +62,8 @@ class EmployeeList extends PolymerElement {
   }
 
   @reflectable
-  void toggleSelection(MouseEvent e, [_]) {
-    var item = new DomRepeatModel.fromEvent(e).item;
+  void toggleSelection(e, [_]) {
+    var item = new DomRepeatModel.fromEvent(e)['item'];
```

Comment:
I guess it is no longer a browser event?

I was just thinking about versioning/breaking changes, but I guess you are using -rc.12, so it doesn't matter. right?